src/Importify/Resolution.hs

```diff
@@ -50,3 +64,15 @@ resolveOneModule m =
         symbolsEnv    = resolve [clearedModule] mempty
         symbols       = concat $ M.elems symbolsEnv
     in symbols
+
+collectUsedQuals :: [ImportDecl SrcSpanInfo] -> [Scoped SrcSpanInfo] -> [ModuleName SrcSpanInfo]
```

Comment:
It's my fault I didn't write _Haddock_ for that :( Good point! Thank you for writing docs for this function as well